docs/version0.41.md

```diff
@@ -94,6 +95,12 @@ With this release, the OpenJ9 VM implementation supports thread memory allocatio
 
 Support for the `getThreadAllocatedBytes MXBean` API is not added on z/OS&reg; platforms in this release.
 
+### The `allow` and `disallow` keywords in the `-Djava.security.manager` system property are ignored
+
+From OpenJDK version 18 onwards, to enable or disable the `SecurityManager` at runtime, you must set the `-Djava.security.manager=allow` or `-Djava.security.manager=disallow` options. OpenJDK versions before version 12 now ignore the `allow` and `disallow` keywords, if specified.
```

Comment:
There is a bit of a disconnect. This could mention that 17 already ignored the keywords and 8, 11 are updated to match.